crates/api/src/lib.rs

```diff
@@ -1548,8 +1558,62 @@ mod test {
 
     fn embed_chronicle_tp() -> EmbeddedChronicleTp {
         chronicle_telemetry::telemetry(None, chronicle_telemetry::ConsoleLogging::Pretty);
+        let mut buf = vec![];
+        Setting {
+            entries: vec![Setting_Entry {
+                key: "chronicle.opa.policy_name".to_string(),
+                value: "allow_transactions".to_string(),
+                ..Default::default()
+            }]
+            .into(),
+            ..Default::default()
+        }
+        .write_to_vec(&mut buf)
+        .unwrap();
+        let setting_id = (
+            chronicle_protocol::settings::sawtooth_settings_address("chronicle.opa.policy_name"),
+            buf,
+        );
+        let mut buf = vec![];
+        Setting {
+            entries: vec![Setting_Entry {
+                key: "chronicle.opa.entrypoint".to_string(),
+                value: "allow_transactions.allowed_users".to_string(),
+                ..Default::default()
+            }]
+            .into(),
+            ..Default::default()
+        }
+        .write_to_vec(&mut buf)
+        .unwrap();
 
-        EmbeddedChronicleTp::new()
+        let setting_entrypoint = (
+            chronicle_protocol::settings::sawtooth_settings_address("chronicle.opa.entrypoint"),
+            buf,
+        );
+
+        let d = env!("CARGO_MANIFEST_DIR").to_owned() + "/../../policies/bundle.tar.gz";
```

Comment:
Curious about the tradeoff here with using rust-embed, anything to know or worry about?

Reply:
It's test only, and I was potentially considering scenarios where we use
different policies to the ones we embed to simulate production behaviour where
you can use non default policies

Reply:
That sounds to be a useful developer tip!